example-scripts/readme.md

```diff
@@ -0,0 +1,5 @@
+# Example Scripts 
+
+In what I think is an increasingly bad decision we decided to allow people to use R scripts to upload data. This folder contains a few simple values
+
+We need to think about what packages are available to these scripts and how to limit them from breaking everything. Can they be run in a way which will not effect the enviroment. 
```

Comment:
You're right: we don't want the users to affect the environment. There isn't really any reason to build this functionality into the tool, actually - any data cleaning scripts can just be run separately beforehand and the outputs saved as csvs (or equivalent). The data only needs to be cleaned once, anyway, and with your recent changes users can more easily upload multiple files at once, so there's no need to combine files in a separate script.

Reply:
This is coming from the code I wrote to try and append more meaningful suffixes when joining dfs together. It looks like I didnt check all the functionality before deciding I'd fixed it. I'll see if I can get it working and if not drop back to the commit which just added x or y etc.

Reply:
I reverted to the old one for now. If we think we need better names we can discuss. Its trickier than I thought to keep track of where the columns came from.